src/main/java/frc/robot/subsystems/intake/IntakeIO.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.intake;
+
+import static edu.wpi.first.units.Units.*;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.units.Voltage;
+import org.littletonrobotics.junction.AutoLog;
+
+public interface IntakeIO {
+
+    IntakeInputsAutoLogged inputs = new IntakeInputsAutoLogged();
+
+    void setAngle(MutableMeasure<Angle> angle);
+
+    void setRollerSpeed(MutableMeasure<Velocity<Angle>> speed);
+
+    void setCenterRoller(MutableMeasure<Velocity<Angle>> speed);
```

Comment:
Set what?

Reply:
changed.

Reply:
381418fb60abfd384af6a39eaa907c7483a6ecf4

Reply:
Please fix the merge conflicts.